cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -1225,6 +1225,103 @@ declare namespace Deno {
     export {}; // only export exports
   }
 
+  /**
+   * @category Telemetry
+   * @experimental
+   */
+  export namespace tracing {
+    /**
+     * Whether tracing is enabled.
+     * @category Telemetry
+     * @experimental
+     */
+    export const enabled: boolean;
+
+    /**
+     * Allowed attribute type.
+     * @category Telemetry
+     * @experimental
+     */
+    export type AttributeValue = string | number | boolean | bigint;
+
+    /**
+     * A tracing span.
+     * @category Telemetry
+     * @experimental
+     */
+    export class Span implements Disposable {
+      readonly traceId: string;
+      readonly spanId: string;
+      readonly parentSpanId: string;
+      readonly kind: string;
+      readonly name: string;
+      readonly startTime: number;
+      readonly endTime: number;
+      readonly status: null | { code: 1 } | { code: 2; message: string };
+      readonly attributes: Record<string, AttributeValue>;
+      readonly traceFlags: number;
+
+      /**
+       * Construct a new Span and enter it as the "current" span.
+       */
+      constructor(
+        name: string,
+        kind?: "internal" | "server" | "client" | "producer" | "consumer",
+      );
+
+      /**
+       * Set an attribute on this span.
+       */
+      setAttribute(
+        name: string,
+        value: AttributeValue,
+      ): void;
+
+      /**
+       * Enter this span as the "current" span.
+       */
+      enter(): void;
+
+      /**
+       * Exit this span as the "current" span and restore the previous one.
+       */
+      exit(): void;
+
+      /**
+       * End this span, and exit it as the "current" span.
+       */
+      end(): void;
+
+      [Symbol.dispose](): void;
+
+      static current(): Span | undefined | null;
+    }
+
+    /**
+     * A SpanExporter compatible with OpenTelemetry.js
+     * @category Telemetry
+     * @experimental
+     */
+    export class SpanExporter {}
+
+    /**
+     * A ContextManager compatible with OpenTelemetry.js
+     * @category Telemetry
+     * @experimental
+     */
+    export class ContextManager {}
+
+    export {}; // only export exports
+  }
+
+  /**
+   * @category Telemetry
+   * @experimental
+   */
+  export namespace metrics {
+    export {}; // only export exports
+  }
```

Comment:
I don't understand this bit - what does `only export exports` mean here?

Reply:
all the namespaces here have this, idk what it does